pkg/util/native/resources.go

```diff
@@ -306,3 +306,7 @@ func MultiplyQuantity(quantity resource.Quantity, y float64) resource.Quantity {
 	value = int64(float64(value) * y)
 	return *resource.NewQuantity(value, quantity.Format)
 }
+
+func ParseQuantityToFloat64(quantity resource.Quantity) float64 {
```

Comment:
why not use AsApproximateFloat64 directly?

Reply:
> why not use AsApproximateFloat64 directly?

All comments have been resolved.